.circleci/config.yml

```diff
@@ -1,24 +1,23 @@
 version: 2.1
 
 orbs:
-  node: circleci/node@4.9.0
   browser-tools: circleci/browser-tools@1.2.4
 
+executors:
+  node_lts:
```

Comment:
**nitpick**: this name won't be true in [less than 5 months](https://nodejs.dev/en/about/releases/) 😄 maybe something more generic like `node` or something more static like `node18_16` would be more future proof?

Reply:
My thinking was to make it clear that our primary version is the LTS version. However this comment has made me think - maybe we should have a more generic name like "node_present", "node_past", and "node_future" (🎄 👩‍🎤 👨‍🎤 🧑‍🎤 joke intended) which makes it clear of what the current supported version is and means that updating the minor version only has to be done on the image. What do you think?

Reply:
I worry that those names are still too dependent on current state of the Node release schedule and will go out of date and be misleading in a few months (well `node_present` and `node_future` would anyway, the past will always be in the past, thankfully)

Reply:
This is me trying to read the future 🔮 So making a figurative note of it for if we come across it in the future 👍